### DISCUSSION TITLE
Gas usage? How to visualize?

### DISCUSSION THREAD

**Discussion by heyudude** [slipx06/sunsynk-power-flow-card#200] [Q&A]
Hi

We have apart from electricity and solar also gas usage to visualize on the dashboard. How can I add this? I tried Aux but it shows Watts and not a custom unit like m3

Thanks for the nice card. Love it
Regard

**Comment by slipx06** [comment]
Hi. Its not an option at this stage but let me see if we can use AUX for this purpose

**Reply by butako** [reply]
I'm looking for the same thing. I just want to have a box or a label somewhere which can display Gas consumption (in kWh). With a bit of experimentation I added using 'aux_load1_extra'.   The load.aux_loads must be 0. Then it displays as a number above the Load icon (which is my home in this case).
